src/app.module.ts

```diff
@@ -15,14 +17,26 @@ import { AppService } from './app.service';
         port: +configService.get('POSTGRES_PORT'),
         username: configService.get('POSTGRES_USER'),
         password: configService.get('POSTGRES_PASSWORD'),
-        database: configService.get('POSTGRES_DB'),
+        database:
+          process.env.NODE_ENV !== 'test'
+            ? configService.get('POSTGRES_DB')
+            : 'auth_test_db',
         autoLoadEntities: true,
         synchronize: true,
       }),
       inject: [ConfigService],
     }),
+    TypeOrmModule.forFeature([User]),
+    PassportModule,
+    JwtModule.registerAsync({
+      useFactory: (configService: ConfigService) => ({
+        secret: configService.get('JWT_SECRET'),
+        signOptions: { expiresIn: '1h' },
```

Comment:
I suggest adding `1h` as environment variable like `JWT_EXPIRES_IN`
```suggestion
        signOptions: { expiresIn: configService.get('JWT_EXPIRES_IN') },
```

Reply:
Resolved on de6cb9a085289a0c6246bed85a39e083298b148f.